admin_guide/install/deploy_router.adoc

```diff
@@ -187,7 +187,39 @@ endif::[]
 You can link:../high_availability.html[set up a highly-available
 router] on your OpenShift cluster using IP failover.
 
-=== Using Wildcard DNS
+[[customizing-the-default-routing-subdomain]]
+
+=== Customizing the Default Routing Subdomain
+
+You can customize the suffix or the default routing subdomain for your
+environment using the
+link:../master_node_configuration.html#master-configuration-files[master
+configuration file] (the *_/etc/openshift/master/master-config.yaml_* file by
+default). The following example shows how you can set the configured suffix to
+*v3.openshift.test*:
+
+.Master Configuration Snippet
+====
+
+----
+routingConfig:
+  subdomain: v3.openshift.test
+----
+====
+
+With the OpenShift master(s) running the above configuration, the
+link:../../architecture/core_concepts/routes.html#route-hostnames[generated host
+name] for the example of a host added to a namespace *my-namespace* would be:
+
+.Generated Host Name
+====
+
+----
+no-route-hostname.my-namespace.v3.openshift.test
```

Comment:
Thanks, fixed here and in another generated host name in the arch topic that was affected. Also changed use of the example "my-namespace" name to instead use "mynamespace", to cut down on the hyphens that could introduce confusion.